pkgs/by-name/ic/icecream-sundae/package.nix

```
@@ -0,0 +1,59 @@
+{
+  cmake,
+  fetchFromGitHub,
+  glib,
+  icecream,
+  lib,
+  libcap_ng,
+  libnl,
+  lzo,
+  meson,
+  ncurses,
+  ninja,
+  nix-update-script,
+  pkg-config,
+  stdenv,
+  zstd,
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
A bit more preferable than rec. Also there's job getting done on making it propagate differently than rec